src/sdk/client.ts

```diff
@@ -85,7 +85,7 @@ export const makeRequestFactory = (
                 Accept: 'application/json',
               },
         });
-        return { response, options };
+        return { response, options, url };
```

Comment:
I think we should just grab the `url` from `options.url` otherwise it'll tack on all the query params. Unless we explicitly want to include that.

Reply:
I like that instead. Just updated!